### ISSUE TITLE
DateUtil does incorrect conversion of UTC time

### ISSUE THREAD

**Issue by christian-lechner** [arangodb/java-velocypack#6]
Hi,

the `com.arangodb.velocypack.internal.util.DateUtil` does incorrect conversion of `java.util.Date` objects. The error is that the format is set to ISO 8601 in UTC but the time zone is not taken into account.
A date, e.g. `Mon Apr 16 17:17:21 CEST 2018` would be converted to `2018-04-16T17:17:21Z`, but correct would be `2018-04-16T15:17:21Z`.

Fix:
```java
SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"); 
formatter.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Best regards

**Comment by mvollmary** [comment]
Thanks for reporting

**Comment by mvollmary** [comment]
Fixed with version 1.0.15
